File: src/main/java/com/musiqueando/domain/Grabacion.java
package com.musiqueando.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Document(collection = "grabaciones")
public class Grabacion {
    @Id
    private String id;
    private String usuarioId;
    private String nombre;
    private String fecha;
    private double duracion;
    private String path;
}

File: app/src/main/java/com/example/hospitalcompose/domain/modelo/Grabacion.kt
package com.example.hospitalcompose.domain.modelo

import kotlinx.serialization.Serializable
import kotlinx.serialization.SerialName

@Serializable
data class Grabacion(
    @SerialName("_id")
    val id: String,
    val usuarioId: String,
    val nombre: String,
    val fecha: String,
    val duracion: Double,
    val path: String
)
